angular/core/models/event_model.coffee

```diff
@@ -34,5 +34,10 @@ angular.module('loomioApp').factory 'EventModel', (BaseModel) ->
     model: ->
       @recordStore[@constructor.eventTypeMap[@eventable.type]].find(@eventable.id)
 
+    markAsRead: ->
+      return unless @sequenceId > @discussion().lastReadSequenceId
+      @remote.postMember @id, 'mark_as_read'
```

Comment:
Only call 'mark_as_read' if the event's sequence id is greater than the existing last read sequence id